packages/@n8n_io/eslint-config/base.js

```diff
@@ -322,6 +322,11 @@ const config = (module.exports = {
 		 */
 		'@typescript-eslint/triple-slash-reference': 'off', // @TECH_DEBT: Enable, disallowing in all cases - N8N-5820
 
+		/**
+		 * https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/naming-convention.md
+		 */
+		'@typescript-eslint/naming-convention': 'off',
```

Comment:
why did we disable this globally?

Reply:
I don't why but after upgrading the lint-related packages eslint started to report `@typescript-eslint/naming-convention` as errors even though the eslint configs were not changed.

I'll give it another shot with removing it from here